sbin/spark-daemon.sh

```diff
@@ -169,6 +169,7 @@ case $option in
       if [[ $(ps -p "$TARGET_ID" -o args=) =~ $command ]]; then
         echo "stopping $command"
         kill "$TARGET_ID"
```

Comment:
Yeah, it occurs to me that this doesn't even guarantee the process was killed, but that the signal was sent. Well, I kind of like `kill ... && rm -f $pid` and would vote for merging that.